openmdao/core/driver.py

```diff
@@ -723,7 +877,7 @@ def get_driver_objective_calls(self):
         int
             Number of objective evaluations made during a driver run.
         """
-        return 0
+        return self.result.model_evals
```

Comment:
The change in terminology makes me wonder if these method names should be deprecated in favor of `get_driver_model_evals` and `get_driver_deriv_evals`.

But then I'm not sure why these public methods need to exist...  they were only used to populate the `opt_result`, which is no longer the case.   They are not documented anywhere for the user...

Reply:
Good point.  Changed them to being deprecated and removed their override in DifferentialEvolutionDriver.